gradle/SourceTemplate/app/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'com.android.application'
+
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        // other classpath definitions here
+        classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
+    }
+}
+
+android {
+    compileSdkVersion 25
```

Comment:
this should be less boiler platey. there is alreadya place available for this

https://github.com/kageiit/android-studio-gradle-test/blob/master/build.gradle#L98